### PR TITLE
fix(core): runtime.load should return void

### PR DESCRIPTION
**Pull request by alexlbr**

### CODE REVIEW

File: packages/core/src/utils/configureMount.ts [hunk 1]
```diff
@@ -48,6 +49,10 @@ export const configureMount: ConfigureMount = ({
         },
       });
 
+      if (appProps && !isObject(appProps)) {
+        throw new Error(`onBeforeMount must retun an object or void`);
```
Comment (AlexTraher) [inline packages/core/src/utils/configureMount.ts:53]: V. Minor - `return`

**Review by AlexTraher — APPROVED** [review]

File: packages/core/src/utils/configureMount.ts (hunk 1 above)
Comment (alexlbr) [reply packages/core/src/utils/configureMount.ts:53]: What do you mean by `return`? if it throws the return won't be executed, will it?

Comment (AlexTraher) [reply packages/core/src/utils/configureMount.ts:53]: haha sorry, I literally meant there was a typo` `retun` -> `return`

Comment (alexlbr) [reply packages/core/src/utils/configureMount.ts]: I see, I'm so good a typos that I don't see them even when someone points them out 😄

**Pull request merged**